VerticalMetrics/README.md

```diff
@@ -28,25 +28,39 @@ This rule can be voided if a font is being upgraded and previously had inconsist
 
 *For brevity, we'll refer to the 3 sets of metrics as Typo, Hhea, Win.* 
 
-### 4. Win Ascent and Win Descent values must be the same as the family's tallest/deepest yMin and yMax bbox values
+### 4. Win Ascent and Win Descent values must be the same as the family's tallest/deepest yMin and yMax bbox values (rounded up)
 The Microsoft [OpenType specification](https://www.microsoft.com/typography/otspec/os2.htm#wa). Recommends the following:
 >If any clipping is unacceptable, then the value should be set to yMax’.
 
 [Overpass issue #33](https://github.com/RedHatBrand/Overpass/issues/33) demonstrates what happens in MS applications when the Win Ascent and Descent are not set correctly.
 
 By changing these values, the line height will be increased in MS applications. This is can lead to very loose line heights if the bbox is exceedingly tall. This mainly occurs in families featuring Vietnamese, Devanagari and Arabic or other tall scripts. To counteract this, we enable [Use Typo Metrics](https://www.microsoft.com/typography/otspec/os2.htm#fss) and set the Typo values to match the previous Win values. By swapping the sets, we should retain the previous line heights in Windows as well as remove the clipping.
 
+We want them to be rounded up to avoid clipping at small sizes.
+
 ### 5. [Use_Typo_Metrics](https://www.microsoft.com/typography/otspec/os2.htm#fss) must be enabled
 This will force MS Applications to use the OS/2 Typo values instead of the Win values. By doing this, we can freely set the Win values to avoid clipping and control the line height with the typo values. It has the added benefit of future line height compatibility. When a new script is added, we simply change the Win values to the new yMin and yMax, without needing to worry if the line height have changed.
 
 ### 6. If the family is being updated, the line height must visually match the previous release.
 Some applications do not allow users to control the line height/leading of their fonts. Word processors and text editors are common culprits. It is essential their documents do not reflow.
 
-### 7. Hhea and Typo metrics should match
+### 7. Hhea and Typo metrics should be equal
 Hhea metrics are used in Mac OS X, whilst Microsoft uses Typo when Use_Typo_Metrics is enabled. They should ideally be identical.
 
 This rule can be voided if a font is being upgraded and previously had inconsistent values.
 
+### 8. LineGap values must be 0
+The LineGap value is a space added to the lineheight created by the union of the (typo/hhea)Ascender and (typo/hhea)Descender. It is handled differently according to the environment. It will be added under the descender height in most desktop apps, will be shared above the Ascender height and under the descender height in web browsers, and ignored in MS Word if use_typo_metrics is enabled. For better linespacing consistentcy accross platforms, (typo/hhea)linegap values must be 0.
```

Comment:
Unless we have solid references or tests which proves this section, I'd avoid discussing it in so much detail

From my limited understanding, the reason why we set the line gaps to 0 is for two reasons:
- The OS/2 win metrics don't have a line gap so when we enable fsSelection bit 7 to use the typo metrics, it makes sense that the sTypoLineGap is set to 0.
- As you pointed out, line gaps are not treated consistently across different apps/backends (I don't know the exact details though)

Reply:
I made an important typo: ignored in MS Word if use_typo_metrics is **disabled**. (mac and windows if I recall but to be confirmed). I forgot, it is also ignored in applications like Indesign.

I already made tests last year. It is also confirmed in the Glyphs tuto stuff about vertical metrics. I can re-do tests if necessary.

Reply:
> I already made tests last year. It is also confirmed in the Glyphs tuto stuff about vertical metrics. I can re-do tests if necessary.

Cool. Can you include a link to it?

Reply:
https://glyphsapp.com/learn/vertical-metrics

Reply:
Thanks. I skimmed the doc and couldn't find any info regarding:

>  It will be added under the descender height in most desktop apps, will be shared above the Ascender height and under the descender height in web browsers

> and ignored in MS Word if use_typo_metrics is enabled

I take it you have tests for this? if so, can you upload them. It would be great to include the glyphsapp link and your tests at the end of this section. It definitely helps make our case stronger. I also doubt the sTypoLineGap is ignored if fsSelection bit 7 is set.

Reply:
As I said, important typo for MS Word, it meant "disabled"
for web fonts, in the section "webfonts strategy": `the line gap is distributed evenly above and below the line.`

I re-do tests and link screenshots here to prove it then.

Reply:
Fabulous and apologies for bringing up the ms word part which you clarified earlier.

Reply:
Change "consistentcy" to "consistently"